### DISCUSSION TITLE
Newly created users can't see servers

### DISCUSSION THREAD

**Discussion by viktorovichandrej** [gravitational/teleport#10041] [Q&A] [answered]
Hello,

I have account in teleport that has one cluster and several SSH connections in Servers. I have also created several users in my account. If I login like newly created user I don't see any servers that are available for main user. How to allow other users connect to servers?

**Comment by klizhentas** [comment]
@viktorovichandrej you should check the roles that are assigned to users. For example, if they have auditor, they might not see or connect to servers. What roles are assigned to them?

**Reply by viktorovichandrej** [reply]
I added all three roles - access, auditor, editor for this user, but I don't see servers.

**Comment by webvictim** [comment, marked as answer]
@viktorovichandrej You'll need to add at least one login to one of those roles. Edit the `access` role via the web UI and change this:

```
spec:
  logins:
  - {{internal.logins}}
```

to something like this:

```
spec:
  logins:
  - {{internal.logins}}
  - ubuntu
```

Then log out and log back in again. You should see servers now.

**Reply by SamuelFoc** [reply]
Hi, I am having the exact same problem. When I open my dashboard I can see just: Resources, Access Requests, and Active Sessions. I already tried your solution (to add ubuntu to logins) but it doesn't work.

**Reply by webvictim** [reply]
@SamuelFoc How did you set up Teleport? What version are you running?

Can you log into the cluster with `tsh login --proxy=<url>` and then run `tctl get nodes`? Does it show anything?

**Reply by SamuelFoc** [reply]
I proceeded exactly according to the docs [here](https://goteleport.com/docs/). However, after the installation, I was not able to log into the server using ssh (key pair) anymore.  When I try to log in using `tsh login --proxy=<url>` I get: 

Get "<url>": dial tcp: lookup https on 127.0.0.53:53: server misbehaving
